src/main/resources/assets/integrateddynamics/lang/en_US.lang

```diff
@@ -807,6 +809,81 @@ operator.operators.integrateddynamics.cast.name=Cast
 operator.operators.integrateddynamics.cast.basename=Cast
 operator.operators.integrateddynamics.cast.tooltip=Cast %s to %s
 
+## Config categories
+config.integrateddynamics.item=Item settings
+config.integrateddynamics.block=Block settings
+config.integrateddynamics.fluid=Fluid settings
+config.integrateddynamics.entity=Entity settings
+config.integrateddynamics.general=General settings
+config.integrateddynamics.worldgeneration=World generation settings
+config.integrateddynamics.enchantment=Enchantment settings
+config.integrateddynamics.mob=Mob settings
+config.integrateddynamics.biome=Biome generation settings
+config.integrateddynamics.machine=Machine settings
+config.integrateddynamics.potion=Potion settings
+config.integrateddynamics.core=Core settings
+config.integrateddynamics.modcompat=Mod compatibility settings
+config.integrateddynamics.capability=Capability settings
+
+# Configuration
+config.integrateddynamics=%s configuration
+
+## Biome settings
+config.integrateddynamics.biomeMeneglin=Enable Meneglin biome
+config.integrateddynamics.biomeMeneglin.spawnWeight=Meneglin biome Spawn Weight
+
+## Block settings
+config.integrateddynamics.menrilLeaves.berriesDropChance=Berries drop chance
+
+## Capability settings
+capability.integrateddynamics.cableFakeable=Fake cables
+capability.integrateddynamics.dynamicLight=Dynamic lighting
+capability.integrateddynamics.dynamicRedstone=Dynamic redstone
+capability.integrateddynamics.energyNetwork=Energy network
+capability.integrateddynamics.facadeable=Facades
+capability.integrateddynamics.networkCarrier=Network carrier
+capability.integrateddynamics.networkElementProvider=Network element provider
+capability.integrateddynamics.partContainer=Part containers
+capability.integrateddynamics.partNetwork=Part network
+capability.integrateddynamics.pathElementProvider=Path element provider
+capability.integrateddynamics.variableContainer=Variable container
+capability.integrateddynamics.variableFacadeHolder=Variable facade holder
+
+## Core settings
+config.integrateddynamics.general.analytics=Send anonymous analytics
+config.integrateddynamics.general.cacheCableModels=Cache cable models
+config.integrateddynamics.general.versionChecker=Enable version check
```

Comment:
Enable version check seems better (and also in the Core itself) that'll be the only one hten